src/System.Numerics.Vectors/tests/GenericVectorTests.cs

```diff
@@ -469,10 +469,10 @@ static GenericVectorTests()
             Vector<T> v1 = new Vector<T>(values1);
             int hash = v1.GetHashCode();
 
-            int expected = 0;
+            int expected = 1;
             for (int g = 0; g < Vector<T>.Count; g++)
             {
-                expected = (((expected << 5) + expected) ^ v1[g].GetHashCode());
+                expected = 31 * expected + v1[g].GetHashCode();
```

Comment:
I'm automatically suspicious of anything which wants to calculate the _expected_ value of the hashcode.  What's the point of it, since the actual value itself is meaningless, and solely an implementation detail?  Isn't it far better to make sure it's stable and respects equality?  (I know the test was already like this, just seems like something that should be cleaned up).